src/options.cpp

```diff
@@ -2086,14 +2088,19 @@ void options_manager::deserialize(JsonIn &jsin)
     while (!jsin.end_array()) {
         JsonObject joOptions = jsin.get_object();
 
-        const std::string name = joOptions.get_string("name");
+        const std::string name = migrateOption( joOptions.get_string("name") );
         const std::string value = joOptions.get_string("value");
-
+        
         add_retry(name, value);
         options[ name ].setValue( value );
     }
 }
 
+std::string options_manager::migrateOption( const std::string &name )
+{
+    return !mMigrateOption[name].empty() ? mMigrateOption[name] : name;
```

Comment:
Please don't query maps like that if the key may not be contained in it. This creates a key/value in the map. Use `map::find` to get an iterator to the existing entry:
```C++
const auto iter = mMigrateOption.find( name );
return iter != mMigrateOption.end() ? iter->second : name;
```
Than you can even make the function const, which it logically should be as it does not change the `option_manager` object.